Makefile-constants.mk

```diff
@@ -1,6 +1,6 @@
 
 ifeq ($(COLLECTOR_BUILDER_TAG),)
-COLLECTOR_BUILDER_TAG=cache
+COLLECTOR_BUILDER_TAG=latest
```

Comment:
I believe the `latest` tag auto-updates to whatever gets pushed to the registry, which could mean that pushing from a PR would "overwrite" the latest tag for the master branch as well, can we double check this?

Reply:
Done